climada/hazard/centroids/test/test_centr.py

```diff
@@ -134,20 +135,48 @@ def test_from_csv(self):
         self.assertEqual(centroids.coord[0][1], 32.57)
         self.assertEqual(centroids.coord[n_centroids - 1][0], -16.23)
         self.assertEqual(centroids.coord[n_centroids - 1][1], 39.91)
+        self.assertEqual(centroids.crs, 'epsg:4326')
 
         Path(tmpfile).unlink()
     
     def test_from_excel(self):
         """Read a centroid excel file correctly."""
-        centroids = Centroids.from_excel(HAZ_TEMPLATE_XLS)
+        tmpfile = Path('test_write_excel.xlsx')
+        test_crs = CRS.from_user_input(4326).to_wkt()
+        sample_data = [
+            [32.57, -25.95, test_crs],
+            [32.46, -25.97],
+            [34.84, -19.83],
+            [39.24, -15.13],
+            [33.47, -19.12],
+            [40.68, -14.56],
+            [36.89, -17.88],
+            [33.58, -16.17],
+            [33.64, -25.04],
+            [36.98, -15.46],
+            [35.34, -23.87],
+            [35.24, -13.3],
+            [40.48, -12.96],
+            [34.73, -19.62],
+            [39.91, -16.23]
```

Comment:
@sarah-hlsn : Why this set of data? Seems hard to read and to check whether it is correct, while also not covering well possible lat/lon values (-180 to 180, -90 to 90).

Reply:
I used a subset of the centroids data from `hazard_template.xlsx`, but I can change this accordingly

Reply:
Yes, that is why they are not well suited for a unit test. Unit tests should be obviously right or wrong, and test reasonable ranges for the values. 
A file instead belongs more to the integration tests where one checks that the methods also work with "real" data. However, when a test fails, one can then not easily know whether new  code is correct or wrong, only that it potentially produces a different result as the old code. Therefore the unit tests are needed that allow to really decide which code is correct.